bedwars-plugin/src/main/java/com/andrei1058/bedwars/money/internal/MoneyPerMinuteTask.java

```diff
@@ -26,6 +26,7 @@ public MoneyPerMinuteTask(Arena arena) {
             for (Player p : arena.getPlayers()) {
                     BedWars.getEconomy ().giveMoney ( p, money );
                     p.sendMessage ( Language.getMsg ( p, Messages.MONEY_REWARD_PER_MINUTE ).replace ( "{money}", String.valueOf ( money ) ) );
+                    Arena.getCoinsEarned().put(p.getUniqueId(), Arena.getCoinsEarned(p.getUniqueId()) + money);
```

Comment:
we should implement a new event like BedWarsMoneyGainEvent and listen to that in otder to track coins earned